jquery.validation/index.d.ts

```diff
@@ -265,31 +285,15 @@ interface JQuery
     removeAttrs(attributes: string): any;
 
     /**
-     * Returns the validations rules for the first selected element
-     */
-    rules(): any;
-
-    /**
-     * Removes the specified rules and returns all rules for the first matched element.
-     *
-     * @param command "remove"
-     * @param rules Removes and returns all rules. Manipulates only rules specified via rules-option or via rules("add").
-     */
-    rules(command: string): any;
-    /**
-     * Removes the specified rules and returns all rules for the first matched element.
-     *
-     * @param command "remove"
-     * @param rules The space-separated names of rules to remove and return. Manipulates only rules specified via rules-option or via rules("add").
-     */
-    rules(command: string, rules: string): any;
-    /**
-     * Adds the specified rules and returns all rules for the first matched element. Requires that the parent form is validated, that is, $("form").validate() is called first
-     *
-     * @param command "add"
-     * @param rules The rules to add. Accepts the same format as the rules-option of the validate-method.
+     * If arguments are ommited returns the validations rules for the first selected element. 
+     * If only "remove" command is supplied all rules are removed.
+     * If "remove" command and rules are supplied, only matched rules are removed.
+     * If "add" command is supplied, rules parameter should be supplied as well. Supplied rules are added or overwrite matching existing rules. 
```

Comment:
It would be better to reduce the number of overloads instead of removing them entirely:

```ts
rules(): any
rules(command: "remove", rules?: string): any
rules(command: "add", JQueryValidation.RulesDictionary): any
```

This way the caller knows 

1. the only allowed commands are 'add' and 'remove' 
2. 'remove' has an optional parameter (space-separated string)
3. 'add' has a required parameter.